blueocean-dashboard/src/main/js/components/Runs.jsx

```diff
@@ -61,61 +62,53 @@ export default class Runs extends Component {
 
         const pipelineName = decodeURIComponent(pipeline);
         const runDetailsUrl = buildRunDetailsUrl(organization, fullName, pipelineName, id, 'pipeline');
-           
-        const open = (event) => {
-            if (event) {
-                event.preventDefault();
-            }
-            location.pathname = runDetailsUrl;
-            router.push(location);
-        };
-        const RunCol = (props) => <td className="tableRowLink">
-            <a onClick={open} href={`${UrlConfig.getJenkinsRootURL()}/blue${runDetailsUrl}`}>{props.children}</a>
-        </td>;
-        
+
         const openRunDetails = (newUrl) => {
             location.pathname = newUrl;
             router.push(location);
         };
-        return (<tr key={id} onClick={open} id={`${pipeline}-${id}`} >
```

Comment:
This click handler was causing a double `router.push` that was breaking the "back" functionality in Run Details (JENKINS-39888)